### PR TITLE
Add the ability of toga.Commands to use shortcuts in Windows

### LINKED ISSUE #991: Keyboard shortcut in tutorial 2 is not working
**Describe the bug**
I was expecting Ctrl+k on Windows 10 to trigger action 3, but it didn't seem to do anything.

**To Reproduce**
1. Go to https://toga.readthedocs.io/en/latest/tutorial/tutorial-2.html
2. Run the code
3. Press `<MOD_1>+k`
4. Notice how the terminal doesn't print anything

**Expected behavior**
Expected `action 3` to br printed.

**Environment:**
 - Operating System: Windows 10
 - Python version: CPython 3.8.3
 - Software versions:
   - Briefcase:
   - Toga: 0.3.0dev22
   - ...

### PR DESCRIPTION
**Pull request by saroad2**
Fixes #991

## PR Checklist:

- [X] All new features have been tested
- [ ] All new features have been documented
- [X] I have read the **CONTRIBUTING.md** file
- [X] I will abide by the code of conduct

### CODE REVIEW

File: src/winforms/toga_winforms/app.py [hunk 1]
```diff
@@ -144,6 +146,7 @@ def run_app(self):
             self.create()
 
             self.native.ThreadException += self.winforms_thread_exception
+            self.interface.main_window._impl.native.KeyUp += self.winforms_key_press
```
Comment (freakboy3742) [inline src/winforms/toga_winforms/app.py:149]: Two questions here:
1) Is keyUp the right signal to catch? Why catch the release, not the press (i.e., keyDown)?
2) The code placement is odd to me. This doesn't affect how the code runs, but putting a keypress hander on the window in between two native app handlers seems out of place.

**Review by freakboy3742 — CHANGES_REQUESTED** [review]
Can't argue that this definitely works; I'm just not 100% convinced that it's the correct implementation. The most obvious manifestation of the problem that I see is that the menu doesn't contain the keyboard shortcut. Compare the way the Commands menu renders in Tutorial 2:

Screen Shot 2020-09-12 at 9 42 56 am

to the way the File menu renders on Notepad:

Screen Shot 2020-09-12 at 9 42 47 am

I would have expected that something like the [ShortcutKeys](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.toolstripmenuitem.shortcutkeys?view=netcore-3.1) property on a menu item would capture both the creation of the annotation in the menu item, plus catching the keypress when contextually appropriate.

**Comment by saroad2** [comment]
Hey Russell, you are absolutely right! The  *ShortcutKeys* property was the right direction to go to.
Fixed that and now it works beautifully!

if you have any other errors you've found, please let me know!

**Review by freakboy3742 — APPROVED** [review]
Awesome - looks great!

**Pull request merged**